src/Monolog/Processor/IntrospectionProcessor.php

```diff
@@ -55,6 +55,14 @@ public function __invoke(array $record)
             return $record;
         }
 
+        /*
+        * http://php.net/manual/en/function.debug-backtrace.php
+        * As of 5.3.6, DEBUG_BACKTRACE_IGNORE_ARGS option was added.
+        * Any version less than 5.3.6 must use the DEBUG_BACKTRACE_IGNORE_ARGS constant value '2'.
+        */
+        if (!defined('DEBUG_BACKTRACE_IGNORE_ARGS')) {
+            define('DEBUG_BACKTRACE_IGNORE_ARGS', 2);   
```

Comment:
A little history... We are forced to use version PHP 5.3.3. Because of this we set in the composer.json file the platform like so...
`    "config": {
        "platform": {
            "php": "5.3.3"
        }
    },`
...which resulted in monolog "version": "1.22.1" getting pullled in. Monolog states that it requires...
`            "require": {
                "php": ">=5.3.0",
                "psr/log": "~1.0"
            },`
...but DEBUG_BACKTRACE_IGNORE_ARGS is not defined until PHP 5.3.6.

Since this really is a PHP version issue and PHP version less than 5.2.0 would create an undefined constant error for PHP_VERSION_ID. I thought trying to simplify the process and only address the issue at hand which is an "undefined constant".

DEBUG_BACKTRACE_IGNORE_ARGS is not defined prior to PHP 5.3.6. So, DEBUG_BACKTRACE_IGNORE_ARGS would not be defined for any other libraries prior to PHP 5.3.6.

For PHP versions greater than or equal to 5.3.6 the constant "DEBUG_BACKTRACE_IGNORE_ARGS" is defined and would not impact other libraries that use the DEBUG_BACKTRACE_IGNORE_ARGS constant.

In addition, DEBUG_BACKTRACE_IGNORE_ARGS has a value of (int) 2 when it is defined either by PHP or through this code.